src/main/core/scheduler/scheduler_policy_type.h

```diff
@@ -7,8 +7,6 @@
 #define SHD_SCHEDULER_POLICY_TYPE_H_
 
 typedef enum {
-    /* one global unlocked priority queue */
-    SP_SERIAL_GLOBAL,
```

Comment:
The reason why there was originally a C `SchedulerPolicyType` enum and a separate Rust `SchedulerPolicy` enum was that the configuration file didn't support specifying the serial global scheduler, but Shadow still needed this type internally. Now that the serial scheduler is removed and the two enums are the same, we could remove the `SchedulerPolicyType` enum and only use the Rust enum (with a `#[repr(C)]`). I don't think it's important for this PR, but just wanted to mention it.